### PR TITLE
Feature/87 login system

### PR DESCRIPTION
**Pull request by AdrianaAC**

### CODE REVIEW

File: examples/vite-react/tsconfig.app.json [hunk 1]
```diff
@@ -23,5 +23,10 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["src"]
+  "include": [
+    "src",
+    "../../packages/lumina-core/src/components/login/login.tsx",
```
Comment (JWebCoder) [inline examples/vite-react/tsconfig.app.json:28]: please remove this new lines

**Review by JWebCoder — CHANGES_REQUESTED** [review]
- A lot is still needed, there should be no global element target on CSS
- All classes need to be prefixed with "lumina_"
- Use the token variables for our list to set the correct colors
- Fixe the change between the multiple screens
- The logic by the isLoggedIn should only affect the editor mode
- Our package does not support css.modules, please use the BEM approach we are doing
- remove the unwanted files
- remove the changes from tsconfig.app.json

**Comment by AdrianaAC** [comment]
> * A lot is still needed, there should be no global element target on CSS - [solved]
> * All classes need to be prefixed with "lumina_" - [solved]
> * Use the token variables for our list to set the correct colors - [solved]
> * Fixe the change between the multiple screens
> * The logic by the isLoggedIn should only affect the editor mode
> * Our package does not support css.modules, please use the BEM approach we are doing - [solved]
> * remove the unwanted files - [solved]
> * remove the changes from tsconfig.app.json - [solved]

**Comment by AdrianaAC** [comment]
Fix the change between the multiple screens - [solved]
The logic by the isLoggedIn should only affect the editor mode - [solved]
navigation now working

To do: 
general components to lumina components
re-use styling

**Comment by AdrianaAC** [comment]
General components replaced by Lumina components.

To do: 
Fix login page styles
Re-use styles
<form> replaced by <Form/> from Lumina

**Comment by AdrianaAC** [comment]
Styles as figma. Styles re-used on all 3 pages. 

To do:
should <form> be a <Form> by Lumina?

**Review by JWebCoder — APPROVED** [review]

**Pull request merged**